src/main/resources/assets/commandblockide/lang/de_de.json

```diff
@@ -0,0 +1,23 @@
+{
+    "arguments.editfunction.tag.unsupported": "Kann keine Funktions Tags bearbeiten",
+    "commands.editfunction.failed.modNotInstalled": "Die Command Block IDE muss installiert sein",
+    "commandBlockIDE.close": "Schließen",
+    "commandBlockIDE.save": "Speichern",
+    "commandBlockIDE.type.redstone": "Impuls",
+    "commandBlockIDE.type.redstoneKonditional": "Impuls, Konditional",
+    "commandBlockIDE.type.auto": "Wiederhohlen",
+    "commandBlockIDE.type.autoKonditional": "Wiederhohlen, Konditional",
```

Comment:
After trying to look this up in a dictionary, this seems like a typo. Should it be “Wiederholen” instead?
Also, see previous comment for `C`/`K` mismatch.

Reply:
You're right, it's "Wiederholen"